### PR TITLE
Add LanguageLink component and tests

### PR DESCRIPTION
**Pull request by liviuignat**
- Added `LanguageLink` component that decides what is the current route and build the language links dynamically
- Add tests

### CODE REVIEW

**Comment by liviuignat** [comment]
TODO: this most probably should be moved to `er-common-components`, we will need it in other places.

File: src/universal/common/components/LanguageLink/__tests__/LanguageLink.test.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```
Comment (RaulTsc) [inline src/universal/common/components/LanguageLink/__tests__/LanguageLink.test.tsx]: Uuu, tests. 👍 👍

**Comment by RaulTsc** [comment]
This looks great!

**Comment by RaulTsc** [comment]
And yes, it will surely need to be moved there so we can reuse it later on.

**Pull request merged**